addons/markers/XEH_postInit.sqf

```diff
@@ -9,9 +9,9 @@
 
 // request marker data for JIP
 if (isMultiplayer && {!isServer} && {hasInterface}) then {
-    private _logic = createGroup sideLogic createUnit ["Logic", [0,0,0], [], 0, "NONE"];
-
-    [QGVAR(sendMarkersJIP), [_logic]] call EFUNC(common,serverEvent);
+    GVAR(localLogic) = sideLogic createUnit ["Logic", [0,0,0], [], 0, "NONE"];
+    GVAR(localGroup) = createGroup GVAR(localLogic);
```

Comment:
Sorry, I just read the doc and you are completely right. I'll change it later today.